app/src/main/java/com/asfoundation/wallet/ui/Erc681ReceiverPresenter.kt

```diff
@@ -20,7 +30,15 @@ internal class Erc681ReceiverPresenter(
   private var disposables: CompositeDisposable,
   private val productName: String?,
   private val partnerAddressService: PartnerAddressService,
+  private val createWebViewPaymentSdkUseCase: CreateWebViewPaymentSdkUseCase,
+  private val isWebViewPaymentFlowUseCase: IsWebViewPaymentFlowUseCase,
+  private val rxSchedulers: RxSchedulers,
+  private val billingAnalytics: BillingAnalytics,
+  private var addressService: AddressService,
+  private val logger: Logger,
 ) {
+  private var firstImpression = true
+  private val TAG = "Erc681ReceiverPresenter"
```

Comment:
```suggestion
  private val TAG = this::class.java.simpleName
```
To avoid wrong namings in the future with refactors.